server/plugin/command.go

```diff
@@ -38,6 +38,27 @@ var validFeatures = map[string]bool{
 	featureStars:         true,
 }
 
+const (
+	subCommandListDeleteAll = "delete-all"
+	subCommandList          = "list"
+	subCommandDelete        = "delete"
+	subCommandAdd           = "add"
+)
```

Comment:
I'm not sure if the code is more readable with these consts.

Reply:
@hanzei this was a suggestion by michael as the key were used in multiple places

Reply:
Can you please link his comment?

Reply:
@hanzei you can refer to this comment https://github.com/mattermost/mattermost-plugin-github/pull/466/files#r720472692

Reply:
@hanzei do you mean to use the value as it is , instead of using this const ?